src/test/java/com/powsybl/openloadflow/util/LoadFlowAssert.java

```diff
@@ -23,6 +23,7 @@
     public static final double DELTA_ANGLE = 1E-6d;
     public static final double DELTA_V = 1E-3d;
     public static final double DELTA_POWER = 1E-3d;
+    public static final double DELTA_I = 1E-2d;
```

Comment:
10e-3 would be better, but as I said before, I have some trouble mapping the Hades2 loadflow with the OpenLoadFlow loadflow to check my results, even on a very small network (4 buses, 1 transfo)